lbz/request.py

```diff
@@ -84,10 +103,48 @@ def json_body(self) -> dict | None:
                 raise BadRequestError(f"Content-Type header is missing or wrong: {content_type}")
         return self._json_body
 
-    def to_dict(self) -> dict:
-        copied = {k: v for k, v in self.__dict__.items() if not k.startswith("_")}
-        copied["headers"] = dict(copied["headers"])
-        copied["user"] = repr(copied["user"])
-        if copied["query_params"] is not None:
-            copied["query_params"] = dict(copied["query_params"])
-        return copied
+
+class WebSocketRequest(BaseRequest):
```

Comment:
Why did you implement this class here, once you created a new dedicated package for websocket things? 🤔